source/numerics.tex

```diff
@@ -2143,7 +2143,7 @@
 \end{itemdescr}
 
 \begin{itemdecl}
-void seed();  
+void A::seed();
```

Comment:
`tuple`, `pair` and `vector` don't use this pattern.

Yeah, maybe drop this commit except the `void` fix. Feel free to open an issue if you like.